meta/store_test.go

```diff
@@ -642,8 +673,9 @@ func TestCluster_Open(t *testing.T) {
 // Store is a test wrapper for meta.Store.
 type Store struct {
 	*meta.Store
-	Listener net.Listener
-	Stderr   bytes.Buffer
+	Listener   net.Listener
+	Stderr     bytes.Buffer
+	LeaveFiles bool
```

Comment:
You can just call `s.Store.Close()` to call the underlying `meta.Store.Close()` instead of the `meta_test.Store.Close()`. Then you don't need this flag.